modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase

```diff
@@ -1177,6 +1177,54 @@ definition {
 			fieldType = "Text");
 	}
 
+	@description = "LPS-184370 - Verify if is possible delete an custom object that is inactive"
+	@priority = 3
+	test CanDeleteAnInactiveObject {
+		task ("Given a custom object with a text field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 184370",
+				objectName = "CustomObject184370",
+				pluralLabelName = "Custom Objects 184370");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Text Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject184370");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject184370");
+		}
+
+		task ("And given an entry is created") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject184370",
+				value = "Entry Test");
+		}
+
+		task ("When the custom object is inactivated") {
+			ObjectAdmin.openObjectAdmin();
+
+			CreateObject.inactiveCustomObject(objectLabel = "Custom Object 184370");
+		}
+
+		task ("And when trying to delete the custom object") {
+			Navigator.gotoBack();
+
+			ObjectAdmin.deleteCustomObjectViaUI(
+				labelName = "Custom Object 184370",
+				objectName = "CustomObject184370");
+		}
+
+		task ("Then the custom object is deleted") {
+			AssertElementNotPresent(
+				key_labelName = "Custom Object 184370",
+				locator1 = "CreateObject#OBJECT_NAME");
+		}
```

Comment:
Combine this into one task:

`Then the inactive object can be deleted`

We don't need to split this into separate tasks since it's part of the same step.